docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+# Use postgres/example user/password credentials
+version: '3.1'
+
+services:
+
+  db:
+    image: postgres
```

Comment:
Generally a good idea to pin to something, so we don't get wrecked on upstream changes. Not a pressing concern.

Reply:
I personally think we shouldn't stick to any specific version unless we have a specific problem with the latest. It isn't a pressing concern indeed so I'd rather leave it like that for now.